ios/authentication.md

```diff
@@ -395,129 +468,174 @@ when configuring authentication using the AWS Amplify CLI.
 ![Image](./images/new-facebook-add-testers.png)
 
 
-11. In the Mobile Hub console, choose `Save changes`.
-
 For more information about integrating with Facebook Login, see the [Facebook Getting Started Guide](https://developers.facebook.com/docs/facebook-login).
 
-## Set Up Google Authentication
+**Amplify CLI Configuration - Facebook**
+
+In a terminal window, navigate to the root of your app files and add the auth category to your app. The CLI prompts you for configuration parameters. Choose **I will setup my own configuration** and **AWS IAM controls** when prompted.
+
+```terminal
+$ cd ./YOUR_PROJECT_FOLDER
+$ amplify add auth
+❯ No, I will set up my own configuration.
+❯ User Sign-Up, Sign-In, connected with AWS IAM controls
+```
+
+Choose **YES** to `? Allow unauthenticated logins?` and **YES** to `? Do you want to enable 3rd party authentication providers in your identity pool?`.
 
-To use the following Google service configuration steps to federate Google as a user sign-in provider for AWS services called in your app, try the AWS Amplify [User Sign-in feature](./add-aws-mobile-user-sign-in).
+**Choose Facebook** and then provide your Facebook **App ID** that you saved earlier.
 
-With AWS Amplify, you can configure a working Google Sign-In feature for both Android and iOS apps. To fully integrate Google Sign-In with your app, AWS Amplify needs information that comes from Google's setup process.
+When configuration for Facebook sign-in is complete, the CLI displays a message confirming that you have configured local CLI metadata for this category. Run the following to update your changes in the cloud:
 
-The following pages detail the Google Sign-In requirements ans steps to integrate Google Sign-In for both iOS and Android apps.
+```terminal
+$ amplify push
+```
 
-* `auth-google-create-google-project` (required for `all apps` regardless of platform)
+You can now [configure Facebook in your mobile app](./authentication#facebook-login-in-your-mobile-app).
 
-* `auth-google-create-oauth-android-clientid` (required for all Android apps)
+Note that the CLI allows you to select more than one identity provider for your app. You can also run `amplify auth update` to add an identity provider to an existing auth configuration.
 
-* `auth-google-create-oauth-ios-clientid` (required for all iOS apps)
+#### Google with Cognito Identity
 
+To federate Google as a user sign-in provider for AWS services called in your app, you will pass tokens to `AWSMobileClient.sharedInstance().federatedSignIn()`. You must first register your application with Google Sign-In and then configure this with Amazon Cognito Identity Pools.
 
-## Add sign-in
+The following sections detail the Google Sign-In requirements ans steps to integrate Google Sign-In for both iOS and Android apps:
```

Comment:
ans => and

Reply:
also should we talk about Android apps  in the iOS docs?

Reply:
I want to review the Android stuff in the second pass with Roshan's update.